### DISCUSSION TITLE
Release Notes in Feed

### DISCUSSION THREAD

**Discussion by ghost** [community/community#13972] [Other Feature Feedback, Questions, & Ideas]
Hey there -- looking at the discussions, it seems as if the "For You" feed is quite divisive :)

I like that I can see the release notes of projects I have stared on the front page (finally, it never worked for me before). It would be great, though, if I could filter only to see Release Notes.

Especially in combination with the new lists feature, it makes sense to star projects for different reasons (my stack, reminder to explore it in the future, ...). 
Release Notes in the feed allow me to quickly see if there are any relevant changes to projects that I should take notice of and may otherwise miss.

Release Notes in For You is an excellent addition from my point of view!

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
